Dockerfiles/syslog-ng/syslog-ng.conf

```diff
@@ -0,0 +1,18 @@
+@version: 3.6
+
+source s_net {
+  tcp(
+     ip("127.0.0.1")
```

Comment:
I think we should use the `0.0.0.0` address to listen on. AFAIK we can't reach the service from outside if it only listens on the loopback address.